File: app/enum/enum.go
package enum

type GamePhase string

const (
	GamePhaseLobby    GamePhase = "GamePhaseLobby"
	GamePhasePregame  GamePhase = "GamePhasePregame"
	GamePhaseStarted  GamePhase = "GamePhaseStarted"
	GamePhaseFinished GamePhase = "GamePhaseFinished"
)

type ClientAction string

const (
	ClientActionDrawRace        ClientAction = "ClientActionRenderRace"
	ClientActionDrawLobby       ClientAction = "ClientActionRenderLobby"
	ClientActionDrawNewPosition ClientAction = "ClientActionDrawNewPosition"
)

type ServerAction string

const (
	ServerActionNameChange             ServerAction = "ServerActionNameChange"
	ServerActionToggleReady            ServerAction = "ServerActionToggleReady"
	ServerActionChooseStartingPosition ServerAction = "ServerActionChooseStartingPosition"
	ServerActionMakeMove               ServerAction = "ServerActionMakeMove"
	ServerActionMoveAnimationDone      ServerAction = "ServerActionMoveAnimationDone"
)

File: lib/src/scripts/enum.ts
export enum GamePhase {
  GamePhaseLobby = "GamePhaseLobby",
  GamePhasePregame = "GamePhasePregame",
  GamePhaseStarted = "GamePhaseStarted",
  GamePhaseFinished = "GamePhaseFinished",
}

export enum ClientAction {
  ClientActionDrawRace = "ClientActionDrawRace",
  ClientActionDrawLobby = "ClientActionDrawLobby",
  ClientActionDrawNewPosition = "ClientActionDrawNewPosition",
}

export enum ServerAction {
  ServerActionNameChange = "ServerActionNameChange",
  ServerActionToggleReady = "ServerActionToggleReady",
  ServerActionChooseStartingPosition = "ServerActionChooseStartingPosition",
  ServerActionMakeMove = "ServerActionMakeMove",
  ServerActionMoveAnimationDone = "ServerActionMoveAnimationDone",
}